src/ontodia/customization/props.ts

```diff
@@ -40,7 +40,8 @@ export interface LinkStyle {
     };
     markerSource?: LinkMarkerStyle;
     markerTarget?: LinkMarkerStyle;
-    labels?: LinkLabelStyle[];
+    label?: LinkLabelStyle;
+    properties?: LinkLabelStyle[];
```

Comment:
The label and properties are separated because if user don't want to change the label, he don't need to add the empty object to the beginning of the array. If don't separate them then the user should always remember that the first object is for the label.